### DISCUSSION TITLE
Can CRPO be implemented in an off-policy manner?

### DISCUSSION THREAD

**Discussion by Zarzard** [PKU-Alignment/omnisafe#264] [Q&A] [answered]
It looks like CRPO can be adapted to algorithms like SAC-Lag, has anyone tried this?

**Comment by Gaiejj** [comment, marked as answer]
CRPO's off-policy version is currently under development. We also welcome community contributions. Feel free to contribute by submitting pull requests to enhance safe reinforcement learning.
